source/includes/_people.md

```diff
@@ -252,14 +252,14 @@ This endpoint searches for people and do not return any email information. To ge
 
 `POST https://api.apollo.io/v1/mixed_people/search`
 
-### Query Parameters
+### Query parameters
 
 Parameter | Description | Example
 --------- | ----------- | -----------
 person_titles (optional) | an array of the person's title. Apollo will return results matching ANY of the titles passed in |  ["sales director", "engineer manager"]
 q_organization_domains (optional) | an array of the company domains to search for, joined by the new line character.  | "google.com\nfacebook.com"
 page (optional) | an integer that allows you to paginate through the results  | 1
```

Comment:
Capitalize the first letter for all description items. It is consistent with other pages